docker/e2e/single-instance.yml

```diff
@@ -98,6 +98,8 @@ services:
       - tinode-0
     image: tinode/exporter:latest
     restart: always
+    ports:
+      - "6222:6222"
```

Comment:
This looks unnecessary given the `"SERVE_FOR": "influxdb"`. It's only useable for prometheus.

Reply:
I'm leaving it open for the status page: it contains a link to the "push" action (in case of influxdb) or "metrics" page (in case of prometheus).